### PR TITLE
Added GLOBALS to ipi/__init__.py, made default float %18.10e

### PR DESCRIPTION
**Pull request by fidanyan**
Here is a clean single-commit implementation with GLOBALS in `ipi/___init__.py`

### CODE REVIEW

**Review by ceriottm — COMMENTED** [review]
I'd call this something like `ipi_global_settings` rather than `GLOBALS` (python has its own __globals__ variable I think) but otherwise, as ugly as it is, I guess it's the best quick-fix for this problem

**Comment by fidanyan** [comment]
Renamed

**Comment by ceriottm** [comment]
are there any examples?

File: ipi/inputs/simulation.py [hunk 1]
```diff
@@ -273,9 +291,10 @@ def fetch(self):
             if k == "system":
                 syslist.append(v.fetch())
             elif k == "system_template":
-                syslist += (
```
Comment (ceriottm) [inline ipi/inputs/simulation.py:276]: Why this change? I think the multi-line comment was more readable

File: ipi/utils/io/inputs/io_xml.py [hunk 2]
```diff
@@ -533,10 +533,10 @@ def write_tuple(data):
 def write_float(data):
     """Writes a formatted string from a float.
 
-    Floats are printed out in exponential format, to 8 decimal places and
-    filling up any spaces under 16 not used with spaces.
+    The default format is exponential to 10 decimal places and
+    filling up any spaces under 18 not used with spaces.
 
-    For example 1.0 --> '  1.00000000e+00'
+    For example 1.0 --> '  1.0000000000e+00'
```
Comment (ceriottm) [inline ipi/utils/io/inputs/io_xml.py:539]: I would do the % 17.9e format so it's space-neutral WRT the current format

**Review by ceriottm — CHANGES_REQUESTED** [review]
I have only those two small comments, and then it'd be good to have this in at least one of the examples, so it's tested and there is an example of using it.

File: ipi/inputs/simulation.py (hunk 1 above)
Comment (fidanyan) [reply ipi/inputs/simulation.py:276]: Not sure what you mean. I noticed that Black exploded a simple expression over many lines because of long comment on the same line, so I moved the comment above the expression and split manually. Mainly to prevent exploding the expression.

File: ipi/utils/io/inputs/io_xml.py (hunk 2 above)
Comment (fidanyan) [reply ipi/utils/io/inputs/io_xml.py:539]: Not sure what you mean. 17.9 isn't neutral with current 16.8 anyways. I guess you mean 16.9 with sacrifice of one whitespace? But worth noticing that there can be a minus sign which takes place of one of two whitespaces. I wouldn't leave only one space because of that.

**Comment by ceriottm** [comment]
ok I misunderstood the diff. I like how you changed it.

**Comment by ceriottm** [comment]
yeah I meant 16.9 with one space separator

**Comment by fidanyan** [comment]
My comment about minus sign holds. But, in general, I feel that you don't like the idea of adding digits into default. TBH, since *.9 is anyways not enough to have meV precision in my system, I don't mind to leave the default 16.8 as it was before - just to stop wasting our time discussing this. And I'll just set 18.10 in my calculations.

Regarding an example - I'll add `floatformat` to one of the fhi-aims examples.

**Review by ceriottm — APPROVED** [review]
